src/cmd-buildextend-live

```diff
@@ -801,7 +856,24 @@ with open(build_semaphore, 'w') as f:
     f.write(f"{time.time_ns()}")
 
 try:
-    generate_iso()
+    if COSA_OSBUILD_LIVEISO == "":
+        generate_iso()
+    else:
+        command = [
+            "/usr/bin/cosa", "supermin-run",
+            "--cache",  "/usr/lib/coreos-assembler/runvm-osbuild",
+            "--config",
+            image_for_disk_json,
+            "--mpp",
+            f"/usr/lib/coreos-assembler/osbuild-manifests/coreos.osbuild.{basearch}.mpp.yaml",
+            "--filepath",
+            tmpisofile
+        ]
+        subprocess.run(command)
+        update_buildmeta()
+        # Extract live artifacts from ISO. OSBUILD does not support an output with multiple files
+        command = ["coreos-installer", "iso", "extract", "pxe", tmpisofile]
```

Comment:
this can be fine for now, but really we should restructure the way we use osbuild today to be able to support multiple outputs. I think the limitation you mention is just part of how we wired in osbuild here.